bundles/org.openhab.binding.myuplink/src/main/java/org/openhab/binding/myuplink/internal/config/MyUplinkConfiguration.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.myuplink.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link MyUplinkConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Alexander Friese - Initial contribution
+ */
+@NonNullByDefault
+public class MyUplinkConfiguration {
+
+    private String clientId = "";
+    private String clientSecret = "";
+
+    private Integer asyncTimeout = 120;
+    private Integer syncTimeout = 120;
```

Comment:
These two do not seem to be configurable; define them as constants.
```suggestion
    private static int ASYNC_TIMEOUT = 120;
    private static int SYNC_TIMEOUT = 120;
```